addons/web/static/src/views/list/list_renderer.js

```diff
@@ -706,7 +706,7 @@ export class ListRenderer extends Component {
         // in those situations, we put the value as title of the cells.
         // This is only necessary for some field types, as for the others, we hardcode
         // a minimum column width that should be enough to display the entire value.
-        if (!(fieldType in FIXED_FIELD_COLUMN_WIDTHS)) {
+        if (!(fieldType in FIXED_FIELD_COLUMN_WIDTHS) && fieldType != "json") {
```

Comment:
Please also edit the preceding comment, as it is no more in sync with what the code really do.

Reply:
done